pkg/api/rest/resttest/resttest.go

```diff
@@ -961,7 +961,7 @@ func (t *Tester) testWatchFields(obj runtime.Object, initWatchFn InitWatchFunc,
 				if !ok {
 					t.Errorf("watch channel should be open")
 				}
-			case <-time.After(time.Millisecond * 100):
+			case <-time.After(time.Second * 30):
```

Comment:
How long do these usually take when they pass?
I guess what I'm concerned about is that this is inside a loop (no sure how many iterations it typically does?), and the fail time for each iteration increases dramatically.  If there's a regression across all of these fields, these unit tests might take many many minutes to fail?